### PR TITLE
Feature/memberships

### PR DESCRIPTION
**Pull request by ergjustin**
Memberships List and Manage

### CODE REVIEW

File: src/pages/memberships/index.vue [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        <v-tabs v-model="tabIndex" grow>
+          <v-tab> View </v-tab>
+          <v-tab> Manage </v-tab>
+
+          <v-tab-item>
+            <v-card outlined tile>
+              <v-container fluid>
+                <v-row dense>
+                  <v-col>
+                    <h2 class="headline">Memberships</h2>
```
Comment (devincowan) [inline src/pages/memberships/index.vue:14]: I think @jscottsf said we want this to be "All Memberships" or similar

**Review by devincowan — CHANGES_REQUESTED** [review]

File: src/components/MembershipsFilter.vue [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  //     this.$emit('handleorganization', organization)
+  //   } else if (personId) {
+  //     const person = this.persons.find(per => per._id === personId)
+  //     this.$emit('handleperson', person)
+  //   }
+
+  //   if (roles) {
+  //     this.$emit('handleRoles', Array.isArray(roles) ? roles : [roles])
+  //   }
+  // }
```
Comment (devincowan) [inline src/components/MembershipsFilter.vue:105]: @ergjustin not sure what you have commented here. Did you leave it intentionally?

File: src/components/MembershipManage.vue [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+      } finally {
+        if (this.selectedPerson) {
+          this.membershipsPersonOuterJoin()
+        } else {
+          this.membershipsOrgOuterJoin()
+        }
+      }
+    },
+
+    async membershipsPersonOuterJoin() {
```
Comment (devincowan) [inline src/components/MembershipManage.vue:286]: nice work!

File: src/plugins/ability.js [hunk 4]
```diff
@@ -55,6 +55,11 @@ export default ({ store }, inject) => {
   inject('canPatch', (name, obj) =>
     store.getters.isAbilityUpdated ? ability.can('patch', as(name, obj)) : false
   )
+  inject('canRemove', (name, obj) =>
```
Comment (devincowan) [inline src/plugins/ability.js:58]: did Scott resolve the issues in the api, for this ability?

**Review by devincowan — APPROVED** [review]
Looks great @ergjustin, thank you. Just some commented code that I'm not sure you intended...

File: src/components/MembershipsFilter.vue (hunk 2 above)
Comment (ergjustin) [reply src/components/MembershipsFilter.vue:105]: This code is for future purpose, for when personId or organizationId in router query
possible: when userdetails page to memberships page

File: src/plugins/ability.js (hunk 4 above)
Comment (ergjustin) [reply src/plugins/ability.js:58]: Not yet, I am planning to delete this ability

Comment (devincowan) [reply src/plugins/ability.js:58]: sounds good

**Comment by devincowan** [comment]
looks good to me @ergjustin

**Pull request merged**